src/rest_api/classes/interaction/widgets/overview.clj

```diff
@@ -69,22 +70,45 @@
 (defn rnai [i]
   {:data (when-let [rhs (:interaction/interaction-rnai i)]
            (for [rh rhs] (pack-obj rh)))
-   :k (keys i)
-   :d (:db/id i)
    :description "RNAi details for the interaction"})
 
-; needs to handle all these types Molecule_regulator|Other_regulator|Other_regulated|Rearrangement|Interactor_overlapping_gene|Feature_interactor
-(defn interactor [i] ;WBInteraction000000001
-  {:data (when-let [hs (:interaction/interactor-overlapping-gene i)]
-           (for [h hs]
-             {:interactor (when-let [gene (:interaction.interactor-overlapping-gene/gene h)]
-                            (pack-obj gene))
-              :interactor_type nil
-              :keys (keys h)
-              :role (when-let [roles (:interactor-info/interactor-type h)]
-                      (for [role roles] (str/capitalize (name role))))
-              :transgene nil
-              :variation nil}))
+; looked for interactors with field variation-interactor and non were filled in by prod
+(defn interactor [i]
+  {:data (not-empty
+           (remove
+             nil?
+           (flatten
+             (conj
```

Comment:
@a8wright not a code style issue, but a reminder that an interactor might also be molecule, pcr, sequence, variation. I'm not sure if all these need to show up. Please double check with curators.